templates/ocs-deployment/filesystem.yaml

```diff
@@ -0,0 +1,54 @@
+#################################################################################################################
+# Create a filesystem with settings with replication enabled for a production environment.
+# A minimum of 3 OSDs on different nodes are required in this example.
+#  kubectl create -f filesystem.yaml
+#################################################################################################################
```

Comment:
If you have used [this file](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/filesystem.yaml) as a base here, please make it clear in the comment.

Also if that is the case, I would even suggest to add the file in it's original form in one commit, and then changing it for the purposes of ocs qe work.